x-pack/test_serverless/functional/services/deployment_agnostic_services.ts

```diff
@@ -34,6 +34,7 @@ const deploymentAgnosticFunctionalServices = _.pick(functionalServices, [
   'dashboardSettings',
   'dashboardVisualizations',
   'dataGrid',
+  'dataStreams',
```

Comment:
> I now see another error
> `Original Error: Error: Unknown Service "dataStreams" on the 3 FTs failing.`

@stratoula this error comes from the agnostic deployment changes (#165887) where the `dataStreams` service was not included in the allowed services. 

@dmlemeshko is there any issue adding the `dataStreams` service here?

Reply:
No issues at all. We decided against "blind" loading of all the stateful FTR services because sometimes serverless has a different logic and requires service changes/re-write. It is really hard to find issues unless we scoped the list of the actually used services.
Feel free to add `dataStreams` and any other ones you need. It is mostly for visibility.

Reply:
It's not an issue to include the `dataStream` service here if it actually works in a serverless environment.